app/controllers/idv/doc_auth_controller.rb

```diff
@@ -11,6 +11,9 @@ class DocAuthController < ApplicationController
 
     before_action :override_document_capture_step_csp
     before_action :update_if_skipping_upload
+    # rubocop:disable Rails/LexicallyScopedActionFilter
```

Comment:
had to disable this cop since `:show` is not explicitly defined on this controller but via magic in the FSM